helm_deploy/laa-dces-drc-integration/templates/ingress.yaml

```diff
@@ -23,39 +26,49 @@ spec:
   tls:
     {{- range .Values.ingress.tls }}
     - hosts:
-        {{- range .hosts }}
+        {{- range .host }}
         - {{ . | quote }}
         {{- end }}
       secretName: {{ .secretName }}
     {{- end }}
   {{- end }}
   rules:
-    {{- range .Values.ingress.hosts }}
-    - host: {{ .host | quote }}
+    - host: {{ .Values.ingress.hosts.mon.host | quote }}
       http:
         paths:
-          {{- range .paths }}
-          - path: {{ . }}
-            pathType: "Prefix"
+          - path: {{ $.Values.actuator.metrics.path }}
+            pathType: "Exact"
             backend:
               service:
                 name: {{ $fullName }}
                 port:
-                  number: {{ $svcPort }}
-          - path: {{ $.Values.actuator.metrics.path }}
+                  number: {{ $.Values.actuator.port }}
+          - path: {{ $.Values.actuator.health.path }}
```

Comment:
If `paths` was still a string array within the `hosts` object array, you could just iterate over them in the array - which could remain in the values file, rather than be half here and half in the values file as this is.

Reply:
Same as above.